src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utilits/_normalize";
+@import "./blocks/header";
+@import "./blocks/_nav";
+@import "./blocks/_catalog";
```

Comment:
u didn't add this files so I cant review them
but one thing, that I note its margin in catalog block
u can see how it looks with margin 0 auto

![image](https://user-images.githubusercontent.com/27231640/221988017-0e76e5d6-ec00-442d-9a8e-e854528cd11b.png)